frontend/controller/actions/identity.js

```diff
@@ -616,14 +619,14 @@ export default (sbp('sbp/selectors/register', {
     await sbp('gi.actions/chatroom/join', {
       ...omit(params, ['options', 'contractID', 'data', 'hooks']),
       contractID: message.contractID(),
-      data: { username: rootState.loggedIn.username }
+      data: {}
     })
 
-    for (const profile of partnerProfiles) {
+    for (const partner of partners) {
       await sbp('gi.actions/chatroom/join', {
         ...omit(params, ['options', 'contractID', 'data', 'hooks']),
         contractID: message.contractID(),
-        data: { username: profile.username }
+        data: { member: partner }
```

Comment:
I think we should use `memberID` everywhere you are using `member` (as an identity contract), and here `partnerID` as well.

Reply:
https://github.com/okTurtles/group-income/pull/1829#discussion_r1481801163